Cargo.toml

```diff
@@ -41,6 +41,10 @@ path = "src/bin/encode.rs"
 name = "binjs_decode"
 path = "src/bin/decode.rs"
 
+[[bin]]
```

Comment:
While you're at it, now that we start to have many binaries, could you add a one-line comment explaining what it does?